game_event/tests.py

```diff
@@ -75,3 +75,27 @@ def test_create_game_event_with_null_ball_game(self, saved_court):
             GameEvent.objects.create(id=TEST_ID, time=TEST_TIME, level_of_game=TEST_LEVEL,
                                      min_number_of_players=TEST_MIN, max_number_of_players=TEST_MAX,
                                      court=saved_court,).full_clean()
+                                     
+
+@pytest.mark.django_db
+class TestGameEventViews:
```

Comment:
I think you also need to add an test 'test_delete_game_event'.
In this case - you need to create an game event, remove it, and try to go again to this url and get a response 404.